### PR TITLE
MAINT-26375 : Add Space Binded Members icon

### PR DESCRIPTION
**Pull request by boubaker**
In Space Members application, an icon is needed to let know more about users that are binded from groups. In addition, this PR will update skeleton of type People list

### CODE REVIEW

File: webapp/portlet/src/main/resources/locale/portlet/social/PeopleListApplication_en.properties [hunk 1]
```diff
@@ -41,3 +41,4 @@ peopleList.label.inviteMembers=Invite members
 peopleList.label.noDataLabel=No data
 peopleList.error.errorWhenSavingSpace=Error after saving a space
 search.connector.label.people=People
+peopleList.label.groupBinded=Group binded
```
Comment (ahamdi) [inline webapp/portlet/src/main/resources/locale/portlet/social/PeopleListApplication_en.properties]:
```suggestion
peopleList.label.groupBound=Group bound
```

File: webapp/portlet/src/main/webapp/people-list/components/PeopleCardFront.vue [hunk 2]
```diff
@@ -24,7 +24,17 @@
         class="peopleInfoIcon d-flex not-clickable primary-border-color ml-1"
         icon
         small>
-        <span class="d-flex uiIconMemberAdmin primary--text"></span>
+        <span v-if="!skeleton" class="d-flex uiIconMemberAdmin primary--text"></span>
+      </v-btn>
+      <v-btn
+        v-if="user.isGroupBound"
+        :title="$t('peopleList.label.groupBinded')"
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/people-list/components/PeopleCardFront.vue]:
```suggestion
        :title="$t('peopleList.label.groupBound')"
```

Comment (boubaker) [reply webapp/portlet/src/main/webapp/people-list/components/PeopleCardFront.vue]: It's the same thing :)
Binded = Bound

Comment (boubaker) [reply webapp/portlet/src/main/webapp/people-list/components/PeopleCardFront.vue]: added anyway ;-)

**Review by ahamdi — APPROVED** [review]

**Pull request merged**